utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -596,8 +596,11 @@ def create_argument_parser():
     option('--libicu', toggle_true('build_libicu'),
            help='build libicu')
 
-    option('--playgroundsupport', store_true('build_playgroundsupport'),
+    option('--playgroundsupport', toggle_true('build_playgroundsupport'),
            help='build PlaygroundSupport')
+    option(['--install-playgroundsupport'],
+           store_true('install_playgroundsupport'),
```

Comment:
I don't think the destination is required in this case since the argument name is the same.

Reply:
Will it auto-substitute the `_` for the `-`?  I thought it doesn't (there were other cases as well).

Reply:
I'll double check, but it should.

Reply:
Actually, I don't think I ever added that convenience here.

Reply:
Okay, so then this I think needs to stay as is.